airbyte-ci/connectors/pipelines/pipelines/commands/groups/tests.py

```diff
@@ -78,4 +78,4 @@ async def run_test(airbyte_ci_package_path: str) -> bool:
         except dagger.ExecError as e:
             logger.error("Tests failed")
             logger.error(e.stderr)
-            return False
+            sys.exit(1)
```

Comment:
Why was this one changed?

Reply:
I figured the `return False` did not lead to a 1 exit code and the test looked passing from a GHA standpoint...